hw-04-weather-api-app/js/app.js

```diff
@@ -0,0 +1,75 @@
+
+/*
+Add a input box to enter city name apply a submit button on the page when user inputs city make sure a function calls openweather api.
+
+when city is entered display results should include city name, current temp, weather description, min temp,max temp.
+
+Once city is picked results should display but also change when adding an if statement to check if the weather is either above 85 degree or below 40.*/
+$(function () {
+  $('#search').click(() => {
+      event.preventDefault()
```

Comment:
@Ashbk32 this should be inside of an event handler such as a `.submit()` see the solution for the way it should be structured: https://github.com/jsd20191008/weather-api-app-solution/blob/master/js/main-async-await.js#L2